marshmallow_recipe/bake.py

```diff
@@ -163,6 +163,13 @@ def post_load(self, data: dict[str, Any]) -> Any:
 
             @m.pre_load  # type: ignore
             def pre_load(self, data: dict[str, Any]) -> Any:
```

Comment:
May be it could be optional (maybe with turn on by default), with an exception for overlap names? This way it might be more useful for general porpoise